_posts/2017-06-09-how-we-found-a-memory-leak.markdown

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title:  "Our journey discovering a Node.js memory leak"
+date:   2017-06-09 17:00:00 +0200
+categories: javascript node.js memoryleak bug
+author: Florian Störkle
+---
+
+# Tracking down a Node.js memory leak
+
+In this blog post we want to share our experience of tracking down and fixing a memory leak in one of our Node.js applications. For several reasons, this proved quite difficult, and took us several days to fix.
+
+## Preface
+
+Our team built a Node.js application that is responsible for synchronizing logs from an external service provider to our internal log pooling system (Google Stackdriver Logs). Let's call this application `sync-logs-to-stackdriver`.
+
+To make it simple, the job periodically checks if there are new logs available in the external system. If that’s the case it syncs them to Stackdriver. If not, it waits a short amount of time and then checks again for new logs.
```

Comment:
Guess that might just be personal preference but I think "that is" is nicer than "that's" here